.circleci/config.yml

```diff
@@ -232,6 +232,39 @@ jobs:
           export NODE_PATH=/usr/lib/node_modules
           validate-template $CIRCLE_WORKING_DIRECTORY/scripts/aws/cloudformation/tasking-manager.template.js
           cfn-config update << parameters.stack_name >> $CIRCLE_WORKING_DIRECTORY/scripts/aws/cloudformation/tasking-manager.template.js -f -c hot-cfn-config -t hot-cfn-config -r $AWS_REGION -p "$JSON_CONFIG"
+    - run:
+        name: Deploy Frontend to S3
+        command: |
+          export TM_APP_BASE_URL=${<< parameters.base_url >>}
+          export TM_APP_API_VERSION=${<< parameters.api_version >>}
+          export TM_CONSUMER_KEY=${<< parameters.consumer_key >>}
+          export TM_CONSUMER_SECRET=${<< parameters.consumer_secret >>}
+          export TM_EMAIL_FROM_ADDRESS=${<< parameters.email_from_address >>}
+          export TM_LOG_DIR=${<< parameters.log_dir >>}
+          export TM_SECRET=${<< parameters.tm_secret >>}
+          export TM_SMTP_HOST=${<< parameters.smtp_host >>}
+          export TM_SMTP_PASSWORD=${<< parameters.smtp_password >>}
+          export TM_SMTP_PORT=${<< parameters.smtp_port >>}
+          export TM_SMTP_USER=${<< parameters.smtp_user >>}
+          export TM_DEFAULT_CHANGESET_COMMENT=${<< parameters.tm_default_changeset_comment >>}
+          export TM_MATOMO_ID=${<< parameters.matomo_site_id >>}
+          export TM_MATOMO_ENDPOINT=${<< parameters.matomo_endpoint >>}
+          export TM_MAPBOX_TOKEN=${<< parameters.mapbox_token >>}
+          export TM_ORG_NAME=${<< parameters.org_name >>}
+          export TM_ORG_CODE=${<< parameters.org_code >>}
+          export TM_ORG_URL=${<< parameters.org_url >>}
+          export TM_ORG_PRIVACY_POLICY=${<< parameters.org_privacy_policy >>}
+          export TM_ORG_TWITTER=${<< parameters.org_twitter >>}
+          export TM_ORG_FB=${<< parameters.org_fb >>}
+          export TM_ORG_INSTAGRAM=${<< parameters.org_instagram >>}
+          export TM_ORG_YOUTUBE=${<< parameters.org_youtube >>}
+          export TM_ORG_GITHUB=${<< parameters.org_github >>}
+          export TM_APP_API_URL="https://`aws cloudformation list-exports --query "Exports[?Name=='tasking-manager-<< parameters.stack_name >>-loadbalancer-url-${AWS_REGION}'].Value" --output=text`"
```

Comment:
@dakotabenjamin could you explain your intention on that part? Will you change it later to the public URL (tasks-stage / tm4 / tasks.hotosm.org)?

Reply:
The frontend uses the public facing URL, not the server. 
![Tasking Manager(1)](https://user-images.githubusercontent.com/1847818/76623270-8644fa80-6509-11ea-8aee-d31e51623733.png)

Reply:
@dakotabenjamin If I would like to use the TM API on another application. Which URL should I use?